diff -U 3 -H -d -r -N -- smb4k-0.6.8.orig/smb4k/core/smb4kcore.cpp smb4k-0.6.8/smb4k/core/smb4kcore.cpp
--- smb4k-0.6.8.orig/smb4k/core/smb4kcore.cpp	2006-02-17 18:23:37.000000000 +0100
+++ smb4k-0.6.8/smb4k/core/smb4kcore.cpp	2006-02-24 18:29:45.000000000 +0100
@@ -235,7 +235,6 @@
   fileList << "ls";
   fileList << "nmblookup";
   fileList << "smbclient";
-  fileList << "net";
 #ifndef __FreeBSD__
   fileList << "mount.cifs";
   fileList << "umount.cifs";
@@ -258,7 +257,8 @@
   optional << "dvips";
   optional << "enscript";
   optional << "rsync";
-
+  optional << "net";
+  
   fileList += optional;
 
   config()->setGroup( "Programs" );
