--- smb4k-0.9.1/smb4k/core/Makefile.am.orig	2007-12-02 11:48:00.000000000 +0100
+++ smb4k-0.9.1/smb4k/core/Makefile.am	2007-12-31 18:21:47.000000000 +0100
@@ -4,13 +4,6 @@
 lib_LTLIBRARIES = libsmb4kcore.la
 libsmb4kcore_la_LDFLAGS = -no-undefined $(all_libraries) -lkwalletclient \
 	-version-info 2:0:0
-include_HEADERS = smb4kauthinfo.h smb4kbookmark.h smb4kbookmarkhandler.h \
-	smb4kcore.h smb4kdefs.h smb4kerror.h smb4kfileio.h smb4kglobal.h \
-	smb4khomesshareshandler.h smb4kmounter.h smb4knetworkitems.h \
-	smb4kpasswordhandler.h smb4kpreviewer.h smb4kpreviewitem.h \
-	smb4kprint.h smb4kprintinfo.h smb4ksambaoptionshandler.h \
-	smb4ksambaoptionsinfo.h smb4kscanner.h smb4kshare.h \
-	smb4ksynchronizationinfo.h smb4ksynchronizer.h
 libsmb4kcore_la_SOURCES = smb4kauthinfo.cpp smb4kbookmark.cpp \
 	smb4kbookmarkhandler.cpp smb4kcore.cpp smb4kerror.cpp smb4kfileio.cpp smb4kglobal.cpp \
 	smb4kglobal_p.cpp smb4khomesshareshandler.cpp smb4kmounter.cpp smb4kmounter_p.cpp \
