--- smb4k-0.8.0/smb4k/core/Makefile.am.orig	2006-10-18 08:00:11.000000000 +0200
+++ smb4k-0.8.0/smb4k/core/Makefile.am	2006-12-22 01:31:58.000000000 +0100
@@ -1,10 +1,5 @@
 lib_LTLIBRARIES = libsmb4kcore.la
 libsmb4kcore_la_LDFLAGS = -no-undefined $(all_libraries) -lkwalletclient -version-info 1:0:0
-include_HEADERS = smb4kglobal.h smb4kauthinfo.h smb4kbookmark.h \
-	smb4kbookmarkhandler.h smb4kcore.h smb4kdefs.h smb4kfileio.h smb4kmounter.h \
-	smb4knetworkitems.h smb4kpasswordhandler.h smb4kprint.h smb4kprintinfo.h smb4kscanner.h \
-	smb4kshare.h smb4kuser.h smb4ksynchronizer.h smb4ksambaoptionshandler.h \
-	smb4khomesshareshandler.h smb4kerror.h smb4ksambaoptionsinfo.h
 libsmb4kcore_la_SOURCES = smb4kauthinfo.cpp smb4kbookmark.cpp \
 	smb4kbookmarkhandler.cpp smb4kcore.cpp smb4kfileio.cpp smb4kmounter.cpp smb4knetworkitems.cpp \
 	smb4kpasswordhandler.cpp smb4kprint.cpp smb4kprintinfo.cpp smb4kscanner.cpp smb4kshare.cpp \
--- smb4k-0.8.0/smb4k/widgets/Makefile.am.orig	2006-09-06 07:02:24.000000000 +0200
+++ smb4k-0.8.0/smb4k/widgets/Makefile.am	2006-12-22 01:32:23.000000000 +0100
@@ -8,13 +8,6 @@
 	smb4ksearchdialog.cpp smb4kshareactionmenu.cpp smb4knetworkitemtooltip.cpp \
 	smb4kbrowseractionmenu.cpp smb4kcustomoptionsdialog.cpp smb4kprintdialog.cpp \
 	smb4kuserinterfaceoptions.cpp
-include_HEADERS = smb4kauthoptions.h smb4kbookmarkeditor.h \
-	smb4kbookmarkmenuwidget.h smb4kbrowserwidget.h smb4kbrowserwidgetitem.h smb4kmountdialog.h \
-	smb4knetworkoptions.h smb4koptionsdlg.h smb4kpreviewdialog.h smb4ksambaoptions.h \
-	smb4kshareoptions.h smb4ksharewidget.h smb4ksuperuseroptions.h smb4ksharewidgetitem.h \
-	smb4ksharesmenuwidget.h smb4ksharetooltip.h smb4krsyncoptions.h smb4ksearchdialog.h \
-	smb4kshareactionmenu.h smb4knetworkitemtooltip.h smb4kbrowseractionmenu.h \
-	smb4kcustomoptionsdialog.h smb4kprintdialog.h smb4kuserinterfaceoptions.h
 INCLUDES = $(all_includes)
 METASOURCES = AUTO
 libsmb4kwidgets_la_LIBADD = $(top_builddir)/smb4k/core/libsmb4kcore.la\
